inventory.yaml

```diff
@@ -2,7 +2,8 @@
 all:
   hosts:
     teamwork-test:
-      ansible_host: teamwork-test
```

Comment:
Why do we change the named ansible-host to the `localhost`?

Reply:
Using 127.0.0.1 ensures consistency and clarity in our playbook, minimizing conflicts for users and providing clear instructions.